src/refactor_nrepl/ns/libspecs.clj

```diff
@@ -110,29 +137,7 @@
   ([ignore-errors? dirs include-tentative-aliases?]
    (let [;; fetch the file list just once (as opposed to traversing the project once for each dialect)
          files (core/source-files-with-clj-like-extension ignore-errors? dirs)
-         ;; pmap parallelizes a couple things:
```

Comment:
This code has been extracted as-is to the new defn, without any creative change.